rest/src/main/java/com/confighub/api/repository/user/property/UpdateKey.java

```diff
@@ -78,6 +84,10 @@ public Response update(@PathParam("account") String account,
             json.addProperty("status", updateStatus.cdr.name());
             json.addProperty("success", true);
 
+            ConcurrentContextPropertiesCache.getInstance().keySet().stream()
+                    .filter(ctx -> ctx.getRepository().equals(repository))
```

Comment:
same as above and for the other files in between - are we just going to cache even if caching isn't enabled and just use the cache when caching is enabled?

Reply:
We don't cache unless the flag is enabled.  These, however, are harmless to leave in all the time.  They are effectively no-ops when caching is disabled.